src/System.IO.Compression.TestData/ZLibTestData/cp.html.z

```diff
@@ -0,0 +1,646 @@
+x`�<head>
+<title>Compression Pointers</title>
```

Comment:
Shouldn’t this file be compressed?

Reply:
I used NoCompression for some, Fastest for some, and Optimal for some.

Reply:
I see.  Mix is fine, I honestly don't know what was done for the other data here, likely it was all compressed at optimal level.  cc @ianhays

Reply:
Yeah, I was just guessing here :-)

Reply:
Perhaps we can fix that by adding a readme that indicates how the files were generated and how they can be regenerated?

Reply:
Sure, added a README.md with the code for a simple C# program to do it.